arm/Microsoft.AppConfiguration/configurationStores/deploy.bicep

```diff
@@ -157,6 +162,18 @@ resource configurationStore 'Microsoft.AppConfiguration/configurationStores@2021
   }
 }
 
+module configurationStore_keyValues 'keyValues/deploy.bicep' = [for (keyValues, index) in keyValuesList: {
+  name: '${uniqueString(deployment().name, location)}-appConfig-KeyValues-${index}'
+  params: {
+    appConfigurationName: configurationStore.name
+    name: keyValues.name
+    value: keyValues.value
+    contentType: contains(keyValues, 'contentType') ? keyValues.contentType : ''
+    tags: contains(keyValues, 'tags') ? keyValues.tags : {}
+    enableDefaultTelemetry: enableReferencedModulesTelemetry
+  }
+}]
+
```

Comment:
```suggestion
module configurationStore_keyValues 'keyValues/deploy.bicep' = [for (keyValue, index) in keyValues: {
  name: '${uniqueString(deployment().name, location)}-appConfig-KeyValues-${index}'
  params: {
    appConfigurationName: configurationStore.name
    name: keyValue.name
    value: keyValue.value
    contentType: contains(keyValue, 'contentType') ? keyValue.contentType : ''
    tags: contains(keyValue, 'tags') ? keyValue.tags : {}
    enableDefaultTelemetry: enableReferencedModulesTelemetry
  }
}]

```

Reply:
In addition, please double check if `contentType` is required or optional. 

- If Required: in the parent deploy.bicep replace `contentType: contains(keyValue, 'contentType') ? keyValue.contentType : ''` with `contentType: keyValue.contentType`. the child deploy.bicep does not need updates.
- If Optional: the parent deploy.bicep can keep the `contentType: contains(keyValue, 'contentType') ? keyValue.contentType : ''`. In the child deploy.bicep the contentType description needs to be changed to 
```
@description('Optional. The content type of the key-values value. Providing a proper content-type can enable transformations of values when they are retrieved by applications.')
param contentType string = ''
```
and the child readme regenerated.

Reply:
Updated deploy.bicep file, the for look now uses keyValues and not keyValuesList. Also verified content type is optional. The readme.md has been updated

Reply:
Thanks @ArielRam99! Almost there. The difference is the loop as follows:
`[for (keyValue, index) in keyValues:`
instead of the current
`[for (keyValues, index) in keyValues:`
Plus `keyValue` and not `keyValues` should be referenced for parameter values.
Replacing the whole block with the above suggestion should solve.

Reply:
Done, next time I'll use copy/paste! Thank you!